.github/workflows/env_containers.yml

```diff
@@ -0,0 +1,19 @@
+name: Build and push base grading containers
+
+on:
+  push:
+    branches: [master]
+
+  workflow_dispatch:
+
+jobs:
+  containers_build_and_push:
+    uses: INGInious/.github/.github/workflows/containers.yml@673723b119b1a92695dd5fea1520f5ae28e38abb
+    with:
+      working-directory: base-containers 
+      context-path: context.yml
+      compose-path: compose.yml
```

Comment:
Where is this file ?

Reply:
It is generated by the [reusable workflow](https://github.com/INGInious/.github/blob/main/.github/workflows/containers.yml) called in the `uses` instruction.